### DISCUSSION TITLE
Error while running volume reconstruction

### DISCUSSION THREAD

**Discussion by askaderd** [PlusToolkit/PlusLib#825] [Q&A]
Hi there,

I'm running into an error and I was wondering if anyone may know what the error is may mean? 

I got this error when sending a .mha sequence file through the volume reconstruction algorithm via Plus command prompt:

Error:
Reading image sequence SlicerIGT-Data\Elbow_output.mha
|ERROR|000.162000| No reader for file: SlicerIGT-Data\Elbow_output.mha| in E:\D\PSNP64b\PlusLib\src\PlusCommon\IO\vtkPlusSequenceIO.cxx(121)
|ERROR|000.195000| Unable to load input sequences file.| in E:\D\PSNP64b\PlusLib\src\PlusVolumeReconstruction\Tools\VolumeReconstructor.cxx(155)

The .mha file is modified from the ElbowUltrasoundSweep.mha (Taken from the SlicerIGT-Data folder) such that the images are replaced with an image sequence of the same resolution (transformation matrices are also replaced). Running the original .mha file through the algorithm works with no errors (following the SlicerIGT-U33 tutorial).

Thanks!
Ammarah Kaderdina

**Comment by lassoan** [comment]
Does everything work with the original input image sequence?

**Reply by askaderd** [reply]
Yes, it works with the original. Viewing the output (from the original sequence) through Slicer, everything looks fine as well.
